Formula/vips.rb

```diff
@@ -0,0 +1,53 @@
+class Vips < Formula
+  desc "Image processing library"
+  homepage "https://github.com/jcupitt/libvips"
+  url "https://github.com/jcupitt/libvips/releases/download/v8.5.5/vips-8.5.5.tar.gz"
+  sha256 "0891af4531d6f951a16ca6d03020b73796522d5fcf7c6247f2f04c896ecded28"
+
+  depends_on "pkg-config" => :build
+  depends_on "fontconfig"
+  depends_on "gettext"
+  depends_on "glib"
+  depends_on "fftw" => :recommended
+  depends_on "giflib" => :recommended
+  depends_on "gobject-introspection" => :recommended
+  depends_on "jpeg" => :recommended
+  depends_on "libexif" => :recommended
+  depends_on "libgsf" => :recommended
+  depends_on "libpng" => :recommended
+  depends_on "librsvg" => :recommended
+  depends_on "libtiff" => :recommended
+  depends_on "little-cms2" => :recommended
+  depends_on "orc" => :recommended
+  depends_on "pango" => :recommended
+  depends_on "poppler" => :recommended
+  depends_on "pygobject3" => :recommended
+  depends_on "python" => :recommended
+  depends_on "graphicsmagick" => :optional
```

Comment:
I think the idea here was that the default configuration would be a set of well-tested file format loaders with a manageable attack surface. If we enable the GM loaders, the number of file formats suddenly shoots up, and many loaders of questionable security are exposed. 

I think most people using vips for web stuff will not want GM enabled.